src/components/util/analytics.js

```diff
@@ -139,9 +139,11 @@ export function initializeGoogleAnalytics(trackingID = null, appOptions = {}){
     ga2('create', trackingID, 'auto');
     ga2(function(tracker){
         const clientID = tracker.get('clientId');
+        console.log("Got client id", clientID);
         if (clientID){
             // Used on backend to associate downloads with user sessions when possible.
-            JWT.cookieStore.set('clientIdentifier', clientID, { path : '/' });
```

Comment:
We no longer have cookieStore (or the universal-cookie npm package, which has somewhat misleading documentation & functionality anyway); is simple enough to just do with raw JS. Maybe could set an expiry time as well, idk.

Reply:
Related - https://developer.mozilla.org/en-US/docs/Web/API/document/cookie